src/AstGen.zig

```diff
@@ -24,9 +24,9 @@ string_bytes: ArrayListUnmanaged(u8) = .{},
 /// would be O(N^2).
 source_offset: u32 = 0,
 /// Tracks the current line of `source_offset`.
-source_line: u32 = 0,
+source_line: u32 = 1,
```

Comment:
I don't think its supposed to start at 1. This messes stuff up for me. I could just add 1, but i'm not sure this is right?